packages/protobuf/package.json

```diff
@@ -35,11 +36,9 @@
     },
     "dependencies": {
         "@trezor/schema-utils": "workspace:*",
-        "long": "^4.0.0",
         "protobufjs": "7.2.6"
     },
     "devDependencies": {
-        "protobufjs-cli": "^1.1.2",
```

Comment:
We need this, it's used in a shell script (`protobuf-build.sh`)

Reply:
fixed in commit: [70b9074](https://github.com/trezor/trezor-suite/pull/12914/commits/70b90749b22d908b7f8860f0972b528bf2dbfdf1)